pkg/arvo/sys/vane/gall.hoon

```diff
@@ -163,10 +155,24 @@
     ?~  res  ~
     `farm(q (~(put by q.farm) i.path u.res))
   ::
+  ++  put-tend
+    |=  [=path =plot]
+    ^-  (unit _farm)
+    ?:  ?=(%coop -.farm)
+      `farm(q (~(put by q.farm) path plot))
```

Comment:
I don't see it. When we traverse the trie we cannot end up at the wrong `%coop` considering that nested `%coop`s are disallowed. Even if the root of the `farm` is a `%coop` that's still the "right %coop".

Reply:
There's no bug here. I'm just a little uncomfortable with how the implementation is factored -- it seems like it could be easy to introduce such a bug in the future. This will do.